packages/cli-kit/src/private/node/session/device-authorization.ts

```diff
@@ -41,7 +41,12 @@ export async function requestDeviceAuthorization(scopes: string[]): Promise<Devi
   })
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  const jsonResult: any = await response.json()
+  let jsonResult: any
+  try {
+    jsonResult = await response.json()
+  } catch (error) {
+    throw new BugError('Received unexpected response. This may be a service outage. Please try again later.')
```

Comment:
Instead of asking them to try again later, could we suggest that they contact [support](https://help.shopify.com/en/contact) instead? (then we may guide folks toward a faster resolution)